uber/site_sections/registration.py

```diff
@@ -41,6 +41,12 @@ def index(self, session, message='', page='1', search_text='', uploaded_id='', o
             count = total_count
 
         attendees = attendees.order(order)
+
+        groups = set()
+        for a in session.query(Attendee).filter(Attendee.first_name == '', Attendee.group_id != None) \
+                                        .options(joinedload(Attendee.group)).all():
+            groups.add((a.group.id, a.group.name+" ("+a.group.leader.full_name+")" or 'BLANK'))
```

Comment:
This raises an exception if we have a group without a leader.

I'll look into what this is doing and make a fix.